docs/Core.rst

```diff
@@ -21,14 +27,51 @@ are used in the same way:
             Most third-party DFHack addons are scripts.
 
 
-.. contents::
+Using DFHack Commands
+=====================
+DFHack commands can be executed in a number of ways:
+
+#. Typing the command into the DFHack console (see below)
+#. From the OS terminal (see below)
+#. Pressing a key combination set up with `keybinding`
+#. From one of several `init-files`, automatically
+#. Using `script` to run a batch of commands from a file
+
+The command line has some nice line editing capabilities, including history
+that's preserved between different runs of DF - use :kbd:`↑` and :kbd:`↓`
+to go through the history.
+
+To include whitespace in the argument/s to some command, quote it in
+double quotes.  To include a double quote character, use ``\"``.
+
+If the first non-whitespace character is ``:``, the command is parsed in
+an alternative mode.  The non-whitespace characters following the ``:`` are
+the command name, and the remaining part of the line is used verbatim as
+the first argument.  This is very useful for the `lua` and `rb` commands.
+As an example, the following two command lines are exactly equivalent::
+
+  :foo a b "c d" e f
+  foo "a b \"c d\" e f"
+
+There are two ways to run DFHack commands from an OS terminal.
+
+* If DF and DFHack are already running, calling ``dfhack-run my command``
+  in an external terminal is equivalent to calling ``my command`` in the
+  DFHack console.  This executable is provided to support external programs,
+  and is not intended for direct use.
+
+* If DF/DFHack is started with arguments beginning with ``+``, the remaining
+  text is treated as a command in the DFHack console.  For example, use
+  `load-save` to skip the main menu with ``./dfhack +load-save region1``.
```

Comment:
Might want to verify that `Dwarf Fortress.exe +load-save region1` works on Windows and document it too.